src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,26 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  comments: Comment[]
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+
+  comments.length
+    ? (
+      <div className="CommentList">
+        {comments.map(comment => <CommentInfo comment={comment} />)}
+      </div>
+    )
```

Comment:
you really need () at here?

Reply:
I've tried but linter throws me an error...